.changeset/smart-jobs-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/svelte": patch
+---
+
+fix: svelte 5 mount/hydrate api change.
```

Comment:
```suggestion
---
"@astrojs/svelte": minor
---

Bumps Svelte 5 peer dependency to `^5.0.0-next.56` and support its latest breaking changes
```

Let's bump this as a minor as its a breaking change but for an experimental feature.